apps/console/src/features/users/pages/users.tsx

```diff
@@ -525,19 +529,108 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
             });
     };
 
+    const addUserDropdownTrigger: ReactElement = (
+        <>
```

Comment:
We can remove the fragment component (<></>), as we already have a parent component.